corehq/apps/reports/tests/test_views.py

```diff
@@ -583,6 +583,10 @@ class TestDomainSharedConfigs(TestCase):
     DOMAIN = 'test_domain'
     OWNER_ID = '5'
 
+    def setUp(self) -> None:
+        # Make sure cached value us cleared
+        ReportConfig.shared_on_domain.clear(ReportConfig, domain=self.DOMAIN)
```

Comment:
I think you need to call super here

Reply:
I left it out because the super class's `setUp` method does not do anything, but for convention's sake I'll add it :+1: